scripts/cmake/CompilerSetup.cmake

```diff
@@ -22,6 +22,12 @@ if(NOT COMPILER_IS_MSVC)
 	set( CMAKE_INCLUDE_SYSTEM_FLAG_CXX "-isystem" CACHE STRING "" FORCE )
 endif()
 
+# When static libraries are going to be used in other libraries position
```

Comment:
"... in other libraries..." means the shared libraries which are dependent on the static libraries.
Comment improvements are welcome.